model/generate.go

```diff
@@ -20,6 +19,14 @@ const (
 	maxGeneratedTasks         = 25000
 )
 
+const (
+	findTasksWithActivationSecondsAttribute  = "evergreen.find_tasks_with_activation.seconds"
+	getTasksWithDependenciesSecondsAttribute = "evergreen.get_tasks_with_dependencies.seconds"
+	addNewTasksSecondsAttribute              = "evergreen.add_new_tasks.seconds"
+	addNewBuildsSecondsAttribute             = "evergreen.add_new_builds.seconds"
+	addDependenciesSecondsAttribute          = "evergreen.add_dependencies.seconds"
+)
+
```

Comment:
Since I'm guessing we'll be doing more of this down the line, I'd advocate against storing the strings fully formed. I think an ideal setup would be where we store the "evergreen" part, the middle part, and the "seconds" part separately, and have a function build the string when we need it. I think that would be more scalable.

Reply:
Oh we actually don't need this block anymore. Removed